### ISSUE TITLE
Remove unneeded values from EventStatus enumerated type to clarify the real-time intent of a WZDx feed

### ISSUE THREAD

**Issue by j-d-b** [usdot-jpo-ode/wzdx#189]
## Background
This issue was "discovered" following discussion about how to represent "recurrent events" in a WZDx feed. Based on several rounds of discussion culminating in a focused meeting on recurrent events in late July 2021, the group came to the consensus that a "recurrent event", that is, a work zone that occurs in a similar fashion multiple days or on a fixed schedule, should be represented with a unique WZDx road event for each occurrence of work. The start and end date time of each road event should reflect the start/end of that occurrence of work. 

In addition, recent discussion with many new members has shown that often individuals don't find WZDx's use case clearly apparent. It is another to do item to update the main project description to clarify the use of a WZDx feed is for making current or upcoming, real time (or as "real time" as possible), work zone information available to the public.

## Proposal
Because of the above discussions, the [EventStatus](https://github.com/usdot-jpo-ode/wzdx/blob/main/spec-content/enumerated-types/EventStatus.md) enumerated type was examined. We determined three of the values—`pending`, `completed`, and `canceled`—should likely be removed. The reasons are as follows:

- `completed` has no utility because completed road events/work zones should not be included in a WZDx feed—a WZDx feed should include real-time information for the traveling public about current and planned road events
- `canceled` does not have utility becuase if a work zone road event is known to be canceled and will no longer occur, it should not be included in a WZDx feed.
- `pending` has a similar functionality to `planned`—they are describing the same concept but the distinction between both is not needed and the descriptions are unnecessarily precise. 

### EventStatus enumerated type after the above changes
Value | Description
--- | ---
`planned` | Indicates the road event is planned to occur in the future. All road events with a start date in the future should have this status. A road event with a date time range encompassing the current date and an `event_status` of `planned` indicates to the consumer that there is likely less accuracy in the road event data or that the work represented by the event hasn't begun yet.
`active` | Indicates that the road event is currently in place and work activity is occurring. It does not signify that there are workers present (see `workers_present` property on the [RoadEvent](https://github.com/usdot-jpo-ode/wzdx/blob/main/spec-content/objects/RoadEvent.md) for that information).

**Comment by sergebeaudry** [comment]
Makes sense

**Comment by j-d-b** [comment]
More thoughts for discussion on this issue after talking with Wisconsin DOT team about how to set EventStatus:

- It is not clear when to set `active` and what the `active` status implies
- Should `active` mean that work is confirmed to be occurring? If so should the value be named `confirmed` instead, and what it means to be confirmed is up to the producer (e.g. through a `status_confirmation_method`, like `location_verify_method`?

As for how event status should is set:

- `active`/`confirmed` could be set automatically when the current datetime matches the road event's `start_date` (lower accuracy that work is actually occurring—this is how Iowa and MassDOT are doing it now)
- `active`/`confirmed` could be set based on deployed ITS equipment (e.g. start/end marker or arrowboard, automatically)
- `active`/`confirmed` could be set on from manual trigger from viewing a camera image or on-site personnel confirmation

**Comment by mark-mockett** [comment]
In discussion of this issue with the Spec. Update co-chairs today, I think we may want to add to the `active` description:  "work activity is occurring *or temporary traffic control is in place*" for example

And it also may be worth adding a also maybe a caveat about confirmation, referencing the "start_date_verification" field as the source for whether the start date has been confirmed from the field. This allows the flexibility for agencies to set a work zone to `active` based on any of the criteria Jacob mentioned above

**Comment by j-d-b** [comment]
After further discussion by the WZDx Specification Update Subgroup Co-chairs, we arrived at a new proposal: **remove the `event_status` property**.

It is easier to determine if work within an event is actively occurring using other existing required properties of the event, specifically, `start_date`, `end_date`, `start_date_accuracy` and `end_date_accuracy` (`event_status` is currently optional). Since in all production WZDx feeds that we know of, `event_status` is set based on the scheduled dates, the `event_status` is not adding any value, just the potential for confusing scenarios.

## Hard to determine scenarios that are currently allowed:

Below are examples of (partial) road event core details that are currently allowed where it is difficult for a producer to determine if the road event has work actively occurring, given the conflict between `event_status`, the current date, and the event's date properties.

Assume for both examples the current date is "2022-01-01 12:00:00" (Jan 1, 2022 at noon).

### Example 1. road event is in the future but has a status of active.

```
{
  "event_status": "active",
  "start_date": "2022-01-15T08:00:00Z",
  "end_date": "2022-01-15T17:00:00Z",
  "start_date_accuracy": "estimated",
  "end_date_accuracy": "estimated"
}
```

### Example 2. current date is within road event dates and dates are verified but the status is planned 

```
{
  "event_status": "planned",
  "start_date": "2022-01-01T08:00:00Z",
  "end_date": "2022-01-01T17:00:00Z",
  "start_date_accuracy": "verified",
  "end_date_accuracy": "verified"
}
```

I think as part of this change that the [TimeVerification](https://github.com/usdot-jpo-ode/wzdx/blob/main/spec-content/enumerated-types/TimeVerification.md) values should be revisited and the descriptions enhanced to clarify that the TimeVerification can be used to determine if work is actively occurring.

**Comment by j-d-b** [comment]
Resolved in #294.
